light_node/Cargo.toml

```diff
@@ -8,11 +8,13 @@ default-run = "light-node"
 [dependencies]
 clap = "2.33"
 dirs = "3.0"
+libc = "0.2.76"
```

Comment:
@michal-stanek 
Do we need this dependency?

Reply:
missed it, thanks!